From 2ae3ee3e6a5880bb67b761484d156c3e05c7776a Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 21 Aug 2017 19:07:57 +0800
Subject: [PATCH 2431/5242] MLK-16242-2 video: fbdev: dcss: correct some irq
 indexes

commit  7850669aee26b8f5d4c6c22c049fe64b2a31c326 from
https://source.codeaurora.org/external/imx/linux-imx.git

The hw irq index for DEC400D, DTRC2 and DTRC3 should
be '15', '16' and '17'. So correct the corresponding
macro definitions in DCSS driver.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/imx_dcss.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/video/fbdev/mxc/imx_dcss.c b/drivers/video/fbdev/mxc/imx_dcss.c
index a78fb27..5672e75 100644
--- a/drivers/video/fbdev/mxc/imx_dcss.c
+++ b/drivers/video/fbdev/mxc/imx_dcss.c
@@ -155,9 +155,9 @@
 #define IRQ_DPR_CH2		4
 #define IRQ_DPR_CH3		5
 #define IRQ_CTX_LD		6
-#define IRQ_DEC400D_CH1		7
-#define IRQ_DTRC_CH2		8
-#define IRQ_DTRC_CH3		9
+#define IRQ_DEC400D_CH1		15
+#define IRQ_DTRC_CH2		16
+#define IRQ_DTRC_CH3		17
 
 /* ctxld irqs status */
 #define RD_ERR			(1 << 16)
-- 
1.7.9.5

